sql/src/main/java/io/crate/analyze/CreateTableAnalyzedStatement.java

```diff
@@ -104,10 +105,14 @@ public boolean isPartitioned() {
         return (Map) mapping().get("properties");
     }
 
-    public List<String> primaryKeys() {
+    public Set<String> primaryKeys() {
```

Comment:
Is this important that it is a set here or could it be exposed as a Collection instead?